recipe/session/sessionFunctions.go

```diff
@@ -139,7 +136,9 @@ func getSessionHelper(recipeImplHandshakeInfo *sessmodels.HandshakeInfo, config
 	requestBody := map[string]interface{}{
 		"accessToken":     parsedAccessToken.RawTokenString,
 		"doAntiCsrfCheck": doAntiCsrfCheck,
-		"enableAntiCsrf":  recipeImplHandshakeInfo.AntiCsrf == antiCSRF_VIA_TOKEN,
+		"enableAntiCsrf":  config.AntiCsrf == antiCSRF_VIA_TOKEN,
+		// TODO NEMI: Dont hardcode
```

Comment:
TODO

Reply:
This is resolved in PR #6: https://github.com/supertokens/supertokens-golang/pull/264

Reply:
Resolved in a future PR